src/duckdb.js

```diff
@@ -208,7 +208,7 @@ async function insertFile(database, name, file, options) {
         }
         if (/\.parquet$/i.test(file.name)) {
           return await connection.query(
-            `CREATE VIEW '${name}' AS SELECT * FROM parquet_scan('${file.name}')`
+            `CREATE VIEW '${name}' AS FROM parquet_scan('${file.name}')`
```

Comment:
Is the shorter form recommended? Seems like the docs still use `SELECT *`.

https://duckdb.org/docs/data/parquet/overview.html

Reply:
It's brand new. Documented in https://duckdb.org/docs/sql/query_syntax/from with one sentence “DuckDB also has an optional FROM-first syntax which enables you to also query without a SELECT statement.” The [announcement](https://duckdb.org/2023/08/23/even-friendlier-sql.html#from-first-in-select-statements) (August 2023) mentions that `SELECT *` is "obsolete".

Reply:
I'll revert that commit for now, since I'm not sure when the new syntax was introduced and it's not related to the FileAttachment issue.